### PR TITLE
Phpbench

### PR DESCRIPTION
**Pull request by Remo**
@mlocati what do you think about this? I've never worked with phpBench before, but it has a nice interface to run method a few times

### CODE REVIEW

File: benchmarks/CalendarBench.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+use Punic\Calendar;
+
+class CalendarBench
+{
+    /**
+     * @Revs(100)
+     * @Iterations(5)
+     */
+    public function benchEraName()
```
Comment (dantleech) [inline benchmarks/CalendarBench.php]: You can move these annotations to the class block:

```php
/**
 * @Revs(10000)
 * @Iterations(10)
 */
class ...
```

Also I would use more revolutions and iterations as indicated above. You might also consider creating an abstract class with these annotations if all your benchmarks have the same requirements.

**Comment by mlocati** [comment]
I never used phpbench... Is this meant to do some benchmarks? In that case, I guess we need some random data, because Punic internally caches some processing...

**Comment by Remo** [comment]
No experience with phpbench either, just started to create some basic benchmarks since I have a few things, not just punic related, where I'd like to improve performance.

**Comment by mlocati** [comment]
Do we still want this?

**Comment by Remo** [comment]
I think it would be nice, but I don't think it's worth the effort.

**Pull request closed without merging**